python/cudf/cudf/_lib/hash.pyx

```diff
@@ -59,17 +59,19 @@ def hash_partition(Table source_table, object columns_to_hash,
     )
 
 
-def hash(Table source_table, object initial_hash_values=None):
+def hash(Table source_table, object initial_hash_values=None, int seed=0):
     cdef vector[uint32_t] c_initial_hash = initial_hash_values or []
     cdef table_view c_source_view = source_table.data_view()
+    cdef int hash_seed = seed
```

Comment:
```suggestion
```
Don't think we are actually using this anywhere?

Reply:
Correct, leftover piece of code that I forgot to delete.